astro/src/content/blog/2024-hacktoberfest.mdx

```diff
@@ -0,0 +1,52 @@
+---
+publish_date: 2024-09-23
+title: Hacktoberfest 2024 with FusionAuth 
+description: FusionAuth joins in the Hacktoberfest event in 2024
+authors: Alex Patterson
+image: /img/blogs/2024-hacktoberfest/hacktoberfest-2024.png
+categories: News
+tags: github, hackathon, open source, hacktoberfest, javascript
+excerpt_separator: "{/* more */}"
+---
+import {RemoteCode} from '@fusionauth/astro-components';
+
+We're excited to announce FusionAuth's participation in this year's Hacktoberfest! Hacktoberfest is a global event that takes place every Autumn, during which developers are encouraged to contribute to open-source projects like FusionAuth. The goal is to foster a vibrant open-source community, celebrate shared knowledge, and make the world of coding more accessible to all. You can find other participating repositories by searching <a href="https://github.com/topics/hacktoberfest" target="_blank" rel="noopener noreferrer">GitHub</a> or <a href="https://gitlab.com/explore/projects/topics/hacktoberfest" target="_blank" rel="noopener noreferrer">GitLab</a>.
+
+{/* more */}
+
+## Getting Started
+
+The first step is to <a href="https://hacktoberfest.com/register" target="_blank" rel="noopener noreferrer"> register for Hacktoberfest</a>. Once you've registered, any PRs submitted to FusionAuth (or any other repo with the label #Hacktoberfest) will count towards your participation in the event!
```

Comment:
Anyone that does not know what a PR is will probably not participate, but may want to spell out pull-request here for those reading.